tasks/sentinel.yml

```
@@ -33,7 +33,7 @@
       paths:
         - ../templates
   register: sentinel_unit_file
-  when: redis_as_service and ansible_service_mgr|default() == "systemd" and not (not redis_install_from_source and redis_service_name == 'redis')
+  when: redis_as_service and ansible_service_mgr|default() == "systemd"
```

I think this actually needs to be 

```suggestion
  when: redis_as_service and ansible_service_mgr|default() == "systemd" and redis_service_name == 'redis')
```

But I was just trying to get this working, and figure out why later.